.github/CODEOWNERS

```
@@ -1 +1 @@
-* @tinymce/integrations-team
\ No newline at end of file
+* @tinymce/team-tinymce
```

This is how we do it in the editor team

```suggestion
* @tinymce/tinymce-reviewers 
```

---

Ok, will update.

---

Done